### PR TITLE
Ecriture de la github action pytest

### PR DESCRIPTION
**Pull request by khun1964**

### CODE REVIEW

**Review by hfdevEX — CHANGES_REQUESTED** [review]
i can't accept this ... sry

**Comment by Riduidel** [comment]
fixes #4

File: .github/workflows/pytest.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest coverage 
+    - name: Install requirements
+      run: pip install -r requirements.txt
+    - name: Build coverage file
+      run: |
+        pytest --cache-clear --cov=app --cov-report xml test/
```
Comment (Riduidel) [inline .github/workflows/pytest.yaml:25]: je pense que tu penses au 📂 dossier `tests`

**Review by Riduidel — CHANGES_REQUESTED** [review]

**Review by Riduidel — APPROVED** [review]

**Review by gpq8l8s — APPROVED** [review]

**Review by hayetyefsah — APPROVED** [review]

**Review by Riduidel — CHANGES_REQUESTED** [review]
no, just no

**Review by Riduidel — APPROVED** [review]

**Pull request merged**